spec/mailers/user_mailer_spec.rb

```diff
@@ -816,6 +823,7 @@ def expect_email_body_to_have_help_and_contact_links
     end
 
     describe '#in_person_ready_to_verify_reminder' do
+      let(:is_enhanced_ipp) { false }
```

Comment:
I think you can remove the is_enhanced_ipp let variable here.

Reply:
Agreed, thanks for catching that.